common/changes/office-ui-fabric-react/persona-coin-initials_2017-11-06-16-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "PersonaCoin: For non-latin initials, show a Contact icon instead of empty string.",
+      "type": "patch"
```

Comment:
Should we rather update the minor version here? Consumers _might_ have expectations on the blank photo in these cases, so making it a minor version is safer

Reply:
I was understanding this as a visual bug, but if partners are relying on the blank photo then I can definitely see what you're saying. Let me ping @dzearing so I can sharpen my understanding.

Reply:
patch vs minor: Will a partner rely on this behavior and be broken if they downgraded? If they won't be broken and it's purely cosmetic, non layout, it's a patch.

Otherwise it's a minor.

Reply:
Thanks David. @mtennoe do you know of cases where customers rely on the behavior of a blank coin? Maybe they are using their own substitutes? Otherwise, seems cosmetic to me.

Reply:
Dont know of any, but just wanted to be sure, so sounds like a patch then :)

Reply:
if there is ambiguity, there is no harm in leaning towards minor. :)

Reply:
I'll go for minor then, sounds like it doesn't hurt!

Reply:
With a minor version, what else do I need to do other than state type: minor in the change file?